tools/update-docker-tags.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+CONSTRAINT=$1
+
+# Using `go run` ensures we are using the version of `update-docker-tags` pinned in `go.mod`
+go run github.com/slimsag/update-docker-tags \
+  -enforce="sourcegraph/cadvisor=$CONSTRAINT" \
+  -enforce="sourcegraph/frontend=$CONSTRAINT" \
+  -enforce="sourcegraph/jaeger-agent=$CONSTRAINT" \
+  -enforce="sourcegraph/github-proxy=$CONSTRAINT" \
+  -enforce="sourcegraph/gitserver=$CONSTRAINT" \
+  -enforce="sourcegraph/grafana=$CONSTRAINT" \
+  -enforce="sourcegraph/indexed-searcher=$CONSTRAINT" \
+  -enforce="sourcegraph/search-indexer=$CONSTRAINT" \
+  -enforce="sourcegraph/jaeger-all-in-one=$CONSTRAINT" \
+  -enforce="sourcegraph/postgres-11.4=$CONSTRAINT" \
+  -enforce="sourcegraph/precise-code-intel-bundle-manager=$CONSTRAINT" \
+  -enforce="sourcegraph/precise-code-intel-worker=$CONSTRAINT" \
+  -enforce="sourcegraph/prometheus=$CONSTRAINT" \
+  -enforce="sourcegraph/query-runner=$CONSTRAINT" \
+  -enforce="sourcegraph/redis-cache=$CONSTRAINT" \
+  -enforce="sourcegraph/redis-store=$CONSTRAINT" \
+  -enforce="sourcegraph/repo-updater=$CONSTRAINT" \
+  -enforce="sourcegraph/searcher=$CONSTRAINT" \
+  -enforce="sourcegraph/symbols=$CONSTRAINT" \
+  base/
```

Comment:
Really not a huge fan that we have to duplicate this across multiple repos now and keep it up to date. Is there a better way we could do that?

Reply:
I wonder why we cannot `-enforce="sourcegraph/.*=$CONSTRAINT"`

Reply:
> Really not a huge fan that we have to duplicate this across multiple repos now and keep it up to date. Is there a better way we could do that?

Yeah, the duplication was unexpected (didn't realize this tool was used in this repo as well) - I think the best home for this script in the future would be in `sourcegraph/sourcegraph`. If we have a "master" list of services somewhere, could generate this script using that as well.

> I wonder why we cannot -enforce="sourcegraph/.*=$CONSTRAINT"

I attempted to add this feature when working on https://github.com/sourcegraph/deploy-sourcegraph/pull/857 , but was unable to:

> Images are defined by hand - I tried an approach with a more generic approach to update all Sourcegraph images, but we have some versioning inconsistencies that made things a bit difficult, so I gave up on that (https://github.com/bobheadlabs/update-docker-tags/commit/38ab3ab34faa97fa18b80a4d3ae76bde9c3d14e9)

I think most notably `sourcegraph/postgres`, etc. had non-`3.x.y` tags which would immediately break a feature like this. Attempts to get around it with regex was extremely painful and I gave up 🙃 Could revisit, though, but I think a better way of maintaining the list of services is better.

Overall, I think the overhead is (hopefully) low, given service additions and removals happen rarely (I think?)

Reply:
What about making it a `buildkite` plugin?

Reply:
This isn't used in buildkite - the one place where it runs in hosted automation is GitHub Actions, and in all other cases it's run on somebody's dev machine (including in the release tooling)